dist/tools/pic32prog/doc.md

```diff
@@ -0,0 +1,136 @@
+Flashing on Linux using pickit-3
+================================
+
+Allow flashing a pic32-wifire from linux using https://github.com/sergev/pic32prog
```

Comment:
Who allows that?

Reply:
Hmm... yeah, maybe this needs a rewording, or just remove "Allow".

Reply:
Flashing another firmware on the pickit-3 allows that.
I could go for "Update the pickit-3 firmware to allow flashing..."

Reply:
(+ build it using `make`)

Reply:
It will be automatically built by the RIOT build system (like edbg).

Reply:
Ahhhh, now I understand the sentence.... the link isn't supposed to be followed, but just used as a noun here... Ok... then maybe use some markdown magic here instead ;-):

```markdown
This document describes how to enable flashing a pic32-wifire from linux using [pic32prog] with a PICkit-3
<!-- .... at the bottom of the document -->
[pic32prog]: https://github.com/sergev/pic32prog
```